test/unit/payload_handler_tests.rb

```diff
@@ -214,8 +214,8 @@ class RouteSpy
     attr_reader :job_passed_to_run, :daemon_data_passed_to_run
     attr_reader :run_called
 
-    def initialize
-      @name = Factory.string
+    def initialize(name)
+      @name = name
```

Comment:
@jcredding maybe switch the reader/etc to be `route_name`?  Again, to help keep straight.